openff/bespokefit/utilities/smirks.py

```diff
@@ -335,12 +335,17 @@ def _get_bespoke_parameters(
             mapped=False, isomeric=False
         ) == fragment.to_smiles(mapped=False, isomeric=False)
 
-        bespoke_smirks: Dict[SMIRKSType, List[str]] = {
-            smirk_type: self._get_bespoke_smirks(
-                parent, fragment, fragment_map_indices, fragment_is_parent, smirk_type
+        bespoke_smirks = []
+        for smirk_type in self.target_smirks:
+            bespoke_smirks.extend(
+                self._get_bespoke_smirks(
+                    parent=parent,
+                    fragment=fragment,
+                    fragment_map_indices=fragment_map_indices,
+                    fragment_is_parent=fragment_is_parent,
+                    smirks_type=smirk_type,
+                )
             )
-            for smirk_type in self.target_smirks
-        }
```

Comment:
Hmm I wonder if this will cause us issues when we come to fit say BCCs or V-Sites that have two parent atoms as these will on the surface look like bond SMIRKS?

Reply:
Good point and this will currently cause an issue with improper vs proper torsions, my other idea was to return the new SMIRKS as [SMIRNOFFParameter](https://github.com/openforcefield/bespoke-fit/blob/cd9749318f73b57494b65a2dd2af9035ccec4e59/openff/bespokefit/schema/smirnoff.py#L257) where the types can be used to avoid this problem.